kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -1,35 +1,157 @@
 <template>
 
-  <div class="fh">
-
+  <CoreBase
+    :immersivePage="false"
+    :immersivePagePrimary="false"
+    :fullScreen="true"
+  >
     <AuthBase>
+      <div v-if="!needsToCreatePassword">
```

Comment:
This makes logical sense, but the fact that the conditional here is so disconnected from the div where we have the create a password logic made it hard for me to read. Don't think refactoring is helpful right now, but maybe just a comment here as well as at the `v-else`, maybe something like:

`Unless we know the user needs to create a password enter this div block for the main flow - see the v-else below for the create password flow`